src/OpenTelemetry/Trace/Export/SimpleSpanProcessor.cs

```diff
@@ -37,14 +45,28 @@ public override void OnStart(Span span)
         /// <inheritdoc />
         public override void OnEnd(Span span)
         {
-            // do not await, just start export
-            this.Exporter.ExportAsync(new[] { span }, CancellationToken.None);
+            try
+            {
+                // do not await, just start export
+                // it can still throw in synchronous part
+                this.Exporter.ExportAsync(new[] { span }, CancellationToken.None);
```

Comment:
Also, to be more explicit, maybe:

```suggestion
                _ = this.Exporter.ExportAsync(new[] { span }, CancellationToken.None);
```